.gitignore

```diff
@@ -42,11 +43,8 @@ inventory_url.yaml
 analytics/config/testFiles/
 analytics/filesystem/testFiles/
 
-# autogenerated version file
-# static/version.txt
-
-.idea/
-.vscode/
+# autogenerated files
+prebid-server.iml
```

Comment:
@Enigo any particular reason for this change. Else could discard this file change

Reply:
cleaned it up a bit:
* `.idea/` was duplicated
* `.vscode/` should be up there with the other folders
* `iml` file is a default IDEA file that should be versioned

Reply:
@onkarvhanumante 
how does it sound to you?

Reply:
or maybe @Sonali-More-Xandr ?

Reply:
> cleaned it up a bit:
> 
> * `.idea/` was duplicated
> * `.vscode/` should be up there with the other folders
> * `iml` file is a default IDEA file that should be versioned

how does above change relates to `Smaato: Add ORTB2.6 tests`

Reply:
it is not related, but the change isn't significant enough to have a dedicated PR, don't you think?

Reply:
Generally its good to isolate 2 different changes in seperate PRs and include details in PR description.

In this case, description was missing for `.idea/` `.vscode/` `iml`

Reply:
alright, the file was reverted